### DISCUSSION TITLE
Change server name on client from IP address to DNS Name

### DISCUSSION THREAD

**Discussion by CountryByte** [Ylianst/MeshCentral#6605] [Q&A]
Is there a bulk way to change the clients Server Connection Name without having to reinstall/update client. I have several clients I installed before I added a Resolvable Name that have IP address. I now have to change the IP Address and need to change that info

**Comment by si458** [comment]
Unfortunately not, u will have to uninstall/redownload/reinstall the meshagent

**Reply by CountryByte** [reply]
Thanks, I was afraid of that from other articles I read.
